frontend/packages/dev-console/integration-tests/features/topology/horizonta_pod_autoscaling/Edit_HPA_Action.feature

```diff
@@ -0,0 +1,73 @@
+Feature: Edit HPA action
+	As a user, I want to edit the HPA assigned to a workload
+ 
+Background:
+    Given user is in developer perspective
+    And user has created a deployment 
+    And user has added CPU resource limit
+    And user has added Memory resource limit
+    And user is in topology
```

Comment:
`And user has created an HPA on this deployment`?

None of the edit flows make sense without this pre-step

Reply:
I've included this in the given statement of the scenarios

Reply:
I understand that, but what's the purpose of a background if not to reduce repeat?